lldb/utils/lit-cpuid/lit-cpuid.cpp

```diff
@@ -20,16 +20,15 @@ using namespace llvm;
 int main(int argc, char **argv) {
 #if defined(__i386__) || defined(_M_IX86) || \
     defined(__x86_64__) || defined(_M_X64)
-  StringMap<bool> features;
-
-  if (!sys::getHostCPUFeatures(features))
+  const StringMap<bool> features = sys::getHostCPUFeatures(features);
+  if (features.empty())
     return 1;
 
-  if (features["sse"])
+  if (features->lookup("sse"))
```

Comment:
Why does this need `->` instead of `.`?

Reply:
It doesn't, I forgot to rebuild on x86. Fixed this and removed the parameter that's no longer needed.